contributing/DOCS-CONTRIBUTING.md

```diff
@@ -68,45 +71,203 @@ documentation efforts, who are happy to help you get started.
+    the Knative Serving component, you might need to add contributor focused
+    information into the `docs` folder of the
+    [knative/serving repo](https://github.com/knative/serving/tree/master/docs/).
+
+  * *Code samples*: Includes contributor related code or samples. Code or samples
+    that are contributor focused also belong in their corresponding
+    component's repo. For example, Eventing specific test code is located in the
+    [knative/eventing tests](https://github.com/knative/eventing/tree/master/test)
+    folder.
+
+* **User focused content**
+
+  * *Documentation*: Includes all content for users of Knative. The
+    external-facing user documentation belongs in the
+    [`knative/docs` repo](https://github.com/knative/docs). All content in
+    `knative/docs` is published to [www.knative.dev](https://www.knative.dev).
+
+  * *Code samples*: Includes user facing code samples and their accompanying
+    step-by-step instructions. User code samples are currently separated into
+    two different locations within the `knative/docs` repo. See the following
+    section for details about determining where you can add your code sample.
+
+##### Determining where to add user focused code samples
+
+There are currently two categories of user focused code samples, either (1)
+*Knative owned and maintained*, or (2) *Community owned and maintained*.
+
+  * **Knative owned and maintained**: Includes code samples that are actively
+    maintained and e2e tested. With currency, clarity, and resource in mind,
+    only the code samples that meet the following requirements are located in
+    the `docs/[*component*]/samples` folders of the `knative/docs` repo:
+
+    * *Actively maintained* - The code sample has an active Knative team member
+      who has committed to regular maintenance of that content and ensures that
+      the code is current and working for every product release.
+    * *Receives regular traffic* - To avoid hosting and maintaining unused
+      or stale content, if code samples are not being viewed and fail to
+      receive attention or use, those samples will be moved into the
+      "community maintained" set of samples.
+    * *Passes e2e testing* - All code samples within `docs/[component]/samples`
+      folders must align with (and pass) the
+      [`e2e` tests](https://github.com/knative/docs/tree/master/test).
+
+    Depending on the Knative component covered by the code sample that you
+    want to contribute, your PR should add that sample in one of the following
+    folders:
+
+    * Build samples: [`/docs/build/samples`](https://github.com/knative/docs/tree/master/docs/build/samples)
+    * Eventing samples: [`/docs/eventing/samples`](https://github.com/knative/docs/tree/master/docs/eventing/samples)
+    * Serving samples: [`/docs/serving/samples`](https://github.com/knative/docs/tree/master/docs/serving/samples)
+
+    Important: If these samples become outdated, receive no traffic, or lack
```

Comment:
This feels repetitive, since it's spelled out so clearly above.

Reply:
i forgot this was here